models/v1beta1/model/model.go

```diff
@@ -100,6 +100,14 @@ type ModelDefinition struct {
 	// To prevent cyclic error, the type is changed to interface, it doesn't affect registration
 	// Edited on purpose
 	Relationships interface{} `json:"relationships" gorm:"-" yaml:"relationships"`
+
+	// Total number of components in a model
+	// Edited on purpose
```

Comment:
What does "Edited on purpose" mean?

Reply:
This file isn't generated from the schema. It's similar to the question I asked on Slack. I believe it was originally generated from the schema and then customized to fit our needs. However, running the schema generation script is overwriting all the changes that were already made to this file.

I suggested updating the JSON schema, as the components and relationship fields in the model file can be handled programmatically during model generation, which would allow us to generate the struct using a script otherwise we are not using benefits of auto struct generation. However, Shlok mentioned that the absence of these fields in model.json is intentional.

I wrote "Edited on purpose" to ensure it is manually updated for now